src/shared/components/stepper/StepperContainer.tsx

```diff
@@ -70,13 +72,20 @@ export const StepperContainer: FC<StepperContainerProps> = ({
                 >
                   BACK
                 </Button>
-                <Button
-                  disabled={!canContinue}
-                  onClick={onLastStep ? submitCallback : onNext}
-                  variant="outlined"
-                >
-                  {onLastStep ? "SUBMIT" : "NEXT"}
-                </Button>
+                {!ready && onLastStep ? (
+                  <ReCAPTCHA
+                    sitekey="6LeFCagaAAAAAM3_suPeXkMLmY77IDsy5f4s-fto"
+                    onChange={(val) => setReady(true)}
+                  />
```

Comment:
Don't know where to put this.. in the UI and the site key!

The token returned by `onChange` should really be sent with the submit for the server to validate but not implemented yet.

Reply:
Smart move putting it where the submit button would be; I think that works.
Not sure about the `sitekey` will have to research that one; and yes, validating on the server is a good idea too